python/paddle/trainer_config_helpers/tests/configs/run_tests.sh

```diff
@@ -16,20 +16,23 @@ if [ -z $1 ]; then
   do
       base_protostr=$protostr/$file
       new_protostr=$protostr/$file.unittest
-      diff $base_protostr $new_protostr -u
+      diff $base_protostr $new_protostr -u &&
+      diff $protostr/$file $protostr/$file.non_file_config.unittest -u
   done
 else
   for file in ${configs[*]}
   do
     if ! $1 $protostr/$file.protostr $protostr/$file.protostr.unittest; then
```

Comment:
Actually in this branch, if statement used for check whether is different between `$protostr/$file.protostr` `$protostr/$file.protostr.unittest`. The rest `diff` command just used to show where is the difference.